src/Uno.UWP/Uno.csproj

```diff
@@ -74,4 +76,24 @@
 	
 	<Target Name="GetBuiltProjectOutputRecursive" Condition=" '$(TargetFramework)' == 'xamarinios10' " />
 
+	<Target Name="VS16Mac_RemoveSystemMemory" BeforeTargets="ResolveAssemblyReferences" Condition="'$(MSBuildVersion)' &gt;= '16.0' and $([MSBuild]::IsOsPlatform('OSX'))">
```

Comment:
This is only needed when building a head, this can be removed.

Reply:
https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=20386&view=logs&jobId=0857e763-0135-5534-86be-f14b16d15a21&j=a5e52b91-c83f-5429-4a68-c246fc63a4f7&t=e7472d04-b09e-59b7-6e25-156b34b46add yes the build fails without that :)